dockers/docker-fpm-frr/bgpd.conf.j2

```diff
@@ -0,0 +1,64 @@
+!
+{% block banner %}
+! =========== Managed by sonic-cfggen DO NOT edit manually! ====================
+! generated by templates/quagga/bgpd.conf.j2 using minigraph_facts.py
+! file: bgpd.conf
+!
+{% endblock banner %}
+!
+{% block system_init %}
+hostname {{ inventory_hostname }}
+password zebra
+log syslog informational
+log facility local4
+! enable password {# {{ en_passwd }} TODO: param needed #}
+{% endblock system_init %}
+!
+{% block bgp_init %}
+!
+! bgp multiple-instance
```

Comment:
should fpm-frr use same configuration as quagga or should it have a separate configuration?